unit_tests/library/route.cpp

```diff
@@ -69,13 +69,13 @@ BOOST_AUTO_TEST_CASE(test_route_same_coordinates_fixture)
                                                    json::Object{{
                                                        {"location", location},
                                                        {"bearing_before", 0},
-                                                       {"bearing_after", 58},
+                                                       {"bearing_after", 238},
```

Comment:
This seems like a strange change ... I wouldn't think that a change to access tags in the profiles would result in different bearings in this tests?

Reply:
Hrm maybe it snaps to a different road now?